clang/lib/Headers/ia32intrin.h

```diff
@@ -361,42 +425,146 @@ _wbinvd(void) {
   __builtin_ia32_wbinvd();
 }
 
+/// Rotates an 8-bit value to the left by the specified number of bits.
+///    This operation is undefined if the number of bits exceeds the size of
+///    the value.
+///
+/// \headerfile <x86intrin.h>
+///
+/// This intrinsic corresponds to the \c ROLB instruction.
```

Comment:
In the Intel and AMD manuals I think the instruction is just called ROL. ROLB is the name in AT&T assembler syntax to distinquish the size.

Reply:
Done